### PR TITLE
add fiscal_comment in view

### PR DESCRIPTION
**Pull request by yk2kus**
add fiscal_comment in customer invoice view, remove field from l10n_br_account_product.

### CODE REVIEW

**Comment by renatonlima** [comment]
Hello @yk2kus 

Can you sign the CLA?

File: l10n_br_account_product/models/account_invoice.py [hunk 1]
```diff
@@ -714,7 +714,6 @@ def _compute_price(self):
         'Outros Custos', digits=dp.get_precision('Account'), default=0.00)
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
-    fiscal_comment = fields.Text(u'Observação Fiscal')
```
Comment (renatonlima) [inline l10n_br_account_product/models/account_invoice.py:717]: :-1: Why remove fiscal_comment field in account.invoice.line?

**Comment by yk2kus** [comment]
Sorry, I misunderstood field with fiscal_comment in 'account.invoice' object in 'l10n_br_account' module that's why it raised error.

**Comment by yk2kus** [comment]
@renatonlima  yes I will sign it today.

**Pull request closed without merging**